OrbitQt/orbitmainwindow.cpp

```diff
@@ -607,6 +610,28 @@ void OrbitMainWindow::ShowCaptureOnSaveWarningIfNeeded() {
   }
 }
 
+void OrbitMainWindow::ShowEmptyFrameTrackWarningIfNeeded() {
+  QSettings settings("The Orbit Authors", "Orbit Profiler");
+  const QString empty_frame_track_warning("EmptyFrameTrackWarning");
+  if (!settings.value(empty_frame_track_warning, false).toBool()) {
+    QMessageBox message_box;
+    message_box.setText(
+        "Frame track enabled for the selected function, but since "
```

Comment:
Do we allow adding multiple frame tracks at once? In that case the warning could be more concrete.

Reply:
You mean adding the function name(s) to the message?

Reply:
For example. I am not too strong on this. If you think it would be fine as it is, also feel free to just resolve the comment.

Reply:
Currently when I select 3 functions in the live tab that have never been called, and click on Enable Frame Track, then I get 3 times the same error message.

Reply:
What do you think about the following change: Add the function name where adding a frame track didn't work and show that in the message. Users can still use the checkbox to not get the second, etc. message.

Showing a single aggregated dialogue requires a bigger logic change.

Reply:
Note that this should be a super-rare error message anyway, as it's not useful to use functions that are not called at a more or less regular frequency.

Reply:
I've added the function name to the message. The message will still pop up multiple times for each function that has 0 hits.

Reply:
Thanks. Sounds good to me.